log/logger.go

```diff
@@ -37,7 +37,7 @@ const (
 	logFilePerm      = os.FileMode(0664)
 
 	// Log file rotation default limits, in bytes.
-	maxLogFileSize   = 5 * 1024 * 1024
```

Comment:
we can keep this  8MB and increase the filecount...also is it tested? like in small size linux and windows vms ?

Reply:
What's the reason behind increasing the filecount while changing the size from 5 to 8?
This is tested; not necessarily on the smallest size VMS. But smallest size VMs have 4GB mem.
https://azure.microsoft.com/en-us/updates/b-series-update-b1ls-is-now-available/
so shouldn;t be a problem